### PR TITLE
update : my wishlist

### PR DESCRIPTION
**Pull request by Bibin-Antony**

### CODE REVIEW

**Comment by Bibin-Antony** [comment]
https://pensive-pike-15b24b.netlify.app/html/wishlist.html 
preview

File: styles/index.css [hunk 1]
```diff
@@ -3,7 +3,11 @@
 @import url('https://fonts.googleapis.com/css2?family=Raleway:wght@100&display=swap');
 @import url(/styles/style.css);
 @import url(/styles/product.css);
+
+@import url(/styles/wishlist.css);
+=======
```
Comment (biswanath-tewari) [inline styles/index.css]: remove this unwanted comment

**Review by biswanath-tewari — CHANGES_REQUESTED** [review]

File: styles/index.css (hunk 1 above)
Comment (Bibin-Antony) [reply styles/index.css]: will do

**Review by biswanath-tewari — APPROVED** [review]

**Review by jeetbhalani15 — APPROVED** [review]
Remove spaces otherwise good to go

**Pull request merged**